src/logic/magicStrangeRoom.ts

```diff
@@ -183,6 +183,12 @@ export class MagicStrageRoom extends LogicBase {
 		this.storyProgress = StoryProgress.beggining;
 		this.lockCode = Math.floor(Math.random() * 9000 + 1000);
 		this.saveSlave = SaveSlave.noSave;
+
+		// Unlock the game's chatroom
+		if (this.conn.chatRoom.Name)
```

Comment:
After the last version you will be able to use `this.conn.isInRoom()`

Reply:
Nice! I updated that check.